Backpacks.cs

```diff
@@ -581,6 +591,38 @@ private bool VerifyCanOpenBackpack(BasePlayer looter, ulong ownerId)
             return true;
         }
 
+        void OnPlayerConnected(BasePlayer player)
+        {
+            CreateGUI(player);
+        }
+
+        void OnPlayerSleepEnded(BasePlayer player)
+        {
+            CreateGUI(player);
+        }
+
+        void CreateGUI(BasePlayer player)
+        {
+            if (player == null || player is NPCPlayer || player is HTNPlayer)
+                return;
```

Comment:
Tip: You can check `player.isNpc` instead of checking both subclasses since they both define `isNpc = true`.

Reply:
Just copied this check from this plugin...
`if (victim is BasePlayer && !(victim is NPCPlayer) && !(victim is HTNPlayer))`

Reply:
`[Error] Error while compiling: Backpacks.cs(609,42): error CS1061: Type `BasePlayer' does not contain a definition for `isNpc' and no extension method `isNpc' of type `BasePlayer' could be found. Are you missing an assembly reference?`